mlir/include/mlir/Dialect/MIGraphX/IR/MIGraphX.td

```diff
@@ -196,9 +196,9 @@ def MIGraphX_QuantizeLinearOp :
 
 def MIGraphX_DeQuantizeLinearOp :
     MIGraphX_Op<"dequantizelinear", [AllElementTypesMatch<["scale", "output"]>]>,
-    Arguments<(ins MIXRShapedOf<[AnyInteger]>:$input,
+    Arguments<(ins MIXRShapedOf<[AnyInteger, F8E4M3FNUZ, F8E5M2FNUZ]>:$input,
```

Comment:
This needs to be `AnyFloat` I see instances where `float` is the input to dequantizelinear, Which is different compared to `onnx` though which doesn't allow float32 for input. 
e.g. 
https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/blob/a09dc50215f91c5ec2c0d6b2afd9fa0ed45f931c/src/simplify_qdq.cpp#L186 

here input to dequantizelinear would be float for the `fp8` convolution/gemms.